### ISSUE TITLE
"vise" is typo'd to "vice" in several translations

### ISSUE THREAD

**Issue by slitherrr** [CleverRaven/Cataclysm-DDA#37087]
# Describe the bug

In these files, there are instances where the word "vice" is used to refer to the tool, for which the generally accepted spelling is "vise":

```
lang/po/ar.po
lang/po/cataclysm-dda.pot
lang/po/da.po
lang/po/de.po
lang/po/el.po
lang/po/es_AR.po
lang/po/es_ES.po
lang/po/fr.po
lang/po/hu.po
lang/po/id.po
lang/po/is.po
lang/po/it_IT.po
lang/po/ja.po
lang/po/ko.po
lang/po/nb.po
lang/po/nl.po
lang/po/pl.po
lang/po/pt_BR.po
lang/po/ru.po
lang/po/sr.po
lang/po/tr.po
lang/po/uk_UA.po
lang/po/zh_CN.po
lang/po/zh_TW.po
```

# Steps To Reproduce

Open the file in text, search for any of the following:

```
msgid "hand vice"
msgid_plural "hand vices
#. ~ Description for hand vice
msgid "This small handheld metal vice is useful for keeping things still."
msgid "Next we need to add some basic tools and vices to the workshop."
```

# Expected behavior

They should all read "vise" instead. "Vice" generally refers to an "immoral behavior", not a tool used to hold things still.

# Versions and configuration

master branch

# Additional context

I made a PR to change this (#37085), but was informed that this is actually performed through Transifex. I signed up to the project to contribute there, but I can't find any UI for changing an _existing_ translation string, only for submitting translations for new strings. My guess is that someone with higher access will have to modify the strings with their facilities for doing so.

**Comment by scorpion451** [comment]
Vice is the correct spelling- it's a weird regional thing in some parts of the US that it's spelled with an s.

Personally my brain refuses to let me use the s spelling, just plain looks wrong and doesn't match the pronunciation (rhymes with ice, not wise).

**Comment by RobertoVGoulart** [comment]
Which of the two spellings is more common in New England? I guess that's the one that the game should use.

**Comment by Qrox** [comment]
You only need to fix strings in the json and cpp files. Source strings in the translation files will be updated through script so manual editing is not needed.

**Comment by MattD94** [comment]
In American English "vise" is the correct spelling, and according to Grammarist.com both forms of spelling have been used interchangeably since it was derived from its Latin root. "Vice" was more or less preferred, but industries in early 19th century America decided to standardize to "vise" in writing so that became the spelling for American English while other forms of English standardized to "vice."

Anyway, to make it consistent with the setting it should probably be "vise" all around.

**Comment by slitherrr** [comment]
@MattD94 has the narrative that agrees with my experience, and Merriam-Webster agrees, noting "vice" in its second entry as "chiefly British spelling of 'vise'" ([source](https://www.merriam-webster.com/dictionary/vice)). I also agree with the determination there that it makes sense for the setting for it to be "vise", and doubly so since all the internal labeling is already "vise" (for the tool quality flag, for example), so it should be consistent from that perspective as well.

**Comment by slitherrr** [comment]
Just to note, by the way, the only instance in the .json or .cpp with this spelling that I found was the one changed in #37085. The other instances (in /lang/po) don't seem to be generated from that. It seems like the base strings generated there live elsewhere?

**Comment by Qrox** [comment]
The strings in /lang/po *are* generated from strings in cpp and json files, but they are only updated periodically, so your changes in #37085 won't automatically update them.

**Comment by slitherrr** [comment]
Sure, but all I mean is, the only change there is a single mention in a description that isn't related at all to any of the strings in /lang/po, and it is the only place where this spelling was used outside of the language file directory, as far as grep could tell.

**Comment by Qrox** [comment]
Oh, now I see what you mean. These strings were already removed from the lang/po files 8 days ago in #37143, so some commit before must have removed them already.

**Comment by ZhilkinSerg** [comment]
Fixed in #37085
